.github/workflows/tests.yml

```diff
@@ -820,6 +815,13 @@ jobs:
           if "${{ github.ref_name }}".startswith("feature/"):
               # feature branch commits are uploaded to a custom label
               envs["ANACONDA_ORG_LABEL"] = "${{ github.ref_name }}"
+
+          elif "${{ github.ref_name }}" == "main":
+              # main branch commits receive the dev label in their version
+              last_tag = check_output(["git", "describe", "--tag", "--abbrev=0"], text=True).strip()
+              num_commits = check_output(["git", "rev-list", "--count", "HEAD", f"{last_tag}..HEAD"], text=True).strip()
+              envs["VERSION_OVERRIDE"] = f"{last_tag}dev{num_commits}"
```

Comment:
this will be versioning all builds after a release as dev builds for that release, so technically backwards

could be very confusing, how concerned are we with this?